src/test/components/MenuButtonsPermalinks.test.js

```diff
@@ -80,6 +80,6 @@ describe('<Permalink>', function() {
       queryInput(),
       'Unable to find the permalink input text field'
     );
-    expect(input.getAttribute('value')).toBe(shortUrl);
+    expect(input).toHaveAttribute('value', shortUrl);
```

Comment:
Do these auto-fix? I get annoyed by the many names of the test functions sometimes, as they are hard to remember. I'd be skeptical of this rule if they don't auto-fix, but if they do, then I guess it's fine. Some of the properties seem nice, and some seem a bit annoying, but if they auto-fix, then the linter will decide for me, which I'm fine with.

Reply:
Yep they all autofix, which was a good surprise! Otherwise I was feeling a bit like you.